src/assets/toolkit/styles/components/thumbnail.css

```diff
@@ -51,7 +51,7 @@
   transition: border 300ms ease; /* 6 */
 }
 
-.Thumbnail:matches(:focus, :hover)::after {
+.Thumbnail:matches(:--enter)::after {
```

Comment:
@tylersticka That is an interesting observation.

The output from that example is:

```css
button,
.button {
  /* styles for your buttons */
}
button:hover,
.button:hover,
button:focus,
.button:focus {
  /*
    hover/focus styles for your button

    Read more about :enter proposal
    http://discourse.specifiction.org/t/a-common-pseudo-class-for-hover-and-focus/877
   */
}
```

So it _does_ appear that when you concatenate a custom selector that itself is a multiple selector, it behaves like `:matches()`.

<img width="1174" alt="screen shot 2016-12-13 at 1 00 58 pm" src="https://cloud.githubusercontent.com/assets/61204/21158988/44ccde6e-c134-11e6-8f72-e1f774a5fdbd.png">

What's unfortunate is how `:not()` behaves a bit differently:

<img width="1139" alt="screen shot 2016-12-13 at 1 04 55 pm" src="https://cloud.githubusercontent.com/assets/61204/21159159/e776609a-c134-11e6-955b-a184f911e130.png">